apps/sample-configs/renderManager.direct.portrait.json

```diff
@@ -6,10 +6,10 @@
         "directModeEnabled": true,
         "directDisplayIndex": 0,
         "directHighPriorityEnabled": true,
-        "numBuffers": 1,
-        "verticalSyncEnabled": false,
+        "numBuffers": 2,
```

Comment:
Getting front-buffer tearing to work would be a remarkable feat.  It requires timing things so that you do all of your rendering and buffer swapping during vertical blanking.  It might some day conceivably work when we're using copy buffers.  Note that double buffering in this context does not provide additional latency -- we're still rendering right before swapping the buffers.